test/test_webdriver.py

```diff
@@ -51,7 +52,7 @@ def test_config_defaults():
     assert wrapper._file_detector is None
     assert wrapper._proxy is None
     assert wrapper._browser_profile is None
-    assert wrapper._options is None
+    assert wrapper._options is not None
```

Comment:
We now should always pass an Options object...whether we're right to, I'm not 100% certain!